styles/stylesheet.css

```diff
@@ -0,0 +1,234 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  /*font-family*/
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  background-image: url("../images/header-image.jpg");
+  padding: 7em;
+  color: #55C632;
+}
+
+.float-right {
```

Comment:
I'd love for you to find another way to layout with out using floats. Consider using flexbox, grid, or position to move these boxes around. Float is just a pain to work with and we want to minimize its use as we have newer tools to use.